### ISSUE TITLE
[a11y] Month heading and Previous and Next buttons do not have roles and labels. 

### ISSUE THREAD

**Issue by DmitriiKasnitskii** [wix/react-native-calendars#1542]
## Description

According WCAG it is necessary to identify page elements properly for screen reader. Implement the month text, for example, “March 2021” as a heading by adding accessibilityRole="heading".

Also there are no accessibilityRole=”button” for Next and Prev buttons attribute to identify the elements as a buttons by default. Provide buttons with accessible names using the accessibilityLabel attribute, for example, accessibilityLabel="Previous month" and accessibilityLabel="Next month". Make sure that on iOS the elements receive focus separately. 
It can be done by Prev/Next customization but there is no roles by default.

## Expected Behavior

The elements are announced as buttons and heading accordingly and have descriptive names.

## Observed Behavior

The elements have neither roles nor accessible names.

**Note:** On iOS the name of the current month and both buttons are implemented as one adjustable control, but changes in months are not announced to the user properly.

## Environment

* `react-native-calendars`: ^1.505.0
* `react-native`: 0.63.2

Also specify:

1. Android 9.0
2. iOS 14.4
